test/modules/viewport.js

```diff
@@ -531,14 +533,14 @@
         var openHandler = function() {
             viewer.removeHandler('open', openHandler);
             var viewport = viewer.viewport;
-            viewport.setRotation(45, true);
+            viewport.setRotation(45, null, true);
             viewport.fitBounds(new OpenSeadragon.Rect(1, 1, 1, 1), true);
             viewport.applyConstraints(true);
             var bounds = viewport.getBounds();
             Util.assertRectangleEquals(
                 assert,
+                new OpenSeadragon.Rect(1.2071067811865466, 0.20710678118654746, Math.sqrt(2), Math.sqrt(2), 45),
```

Comment:
@pearcetm With https://github.com/openseadragon/openseadragon/pull/2249, I suspect you may need to update these numbers. Please take a look!

Reply:
See https://github.com/openseadragon/openseadragon/pull/2249 for fix